File: python/child2/File2.py
from python.child1.File1 import File1


class File2:

    def main(self):
        test = File1("Tes package!")
        print(test.get_data())

if __name__ == "__main__":
    file2 = File2()
    file2.main()


# TO RUN 
# NAVIGATE TO THE DIRECTORY CONTAINING THE "python" directory
# IN THIS CASE "snippets-PBO-02"
# THEN RUN USING "python -m python.child2.File2"

File: cpp/child2/File2.cpp
#include <iostream>
#include "../child1/file1.cpp"
using namespace std;

int main() {
    File1 test = File1("Tes package!");
    cout << test.getData();
}


// TO RUN
// compile using "g++ File2.cpp -o File2" 
// then execute using ".\File2"


